packages/eas-cli/package.json

```diff
@@ -13,7 +13,7 @@
     "@expo/apple-utils": "0.0.0-alpha.20",
     "@expo/config": "3.3.42",
     "@expo/config-plugins": "2.0.2",
-    "@expo/eas-build-job": "0.2.35",
+    "@expo/eas-build-job": "0.2.36",
```

Comment:
Bumped to include https://github.com/expo/eas-build/commit/bf1142132293b6f040d1f6cf83112cc93dc55d4d
<img width="491" alt="Screen Shot 2021-06-16 at 9 01 15 AM" src="https://user-images.githubusercontent.com/1220444/122253868-6aa17000-ce81-11eb-91bc-4d8215465b22.png">